docs/contributing.md

```diff
@@ -1,5 +1,12 @@
+---
```

Comment:
I think it could be valuable to keep a `CONTRIBUTING.md` file in addition to `docs/contributing.md` since it will show up in the GitHub UI, pointing folks towards the contribution guidelines.